### PR TITLE
[W3W, Sign] X-Platform e2e protocol test - session settle

### PR DESCRIPTION
**Pull request by llbartekll**
# Description
passed tests - https://github.com/WalletConnect/WalletConnectSwiftV2/actions/runs/5795589196

Resolves # (issue)

## Due Dilligence

* [ ] Breaking change
* [ ] Requires a documentation update

### CODE REVIEW

File: Example/IntegrationTests/XPlatform/Web3Wallet/XPlatformW3WTests.swift [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                while jsSession == nil {
+                    print("🎃 geting session")
+                    do {
+                        jsSession = try await javaScriptAutoTestsAPI.getSession(topic: session.topic)
+                    } catch {
+                        print("No session on JS client yet")
+                    }
+
+                    if jsSession == nil {
+                        sleep(1)
```
Comment (radeknovis) [inline Example/IntegrationTests/XPlatform/Web3Wallet/XPlatformW3WTests.swift:104]: This is ok for now, but in future, if we are going to have more tests using the JS API, can they implement something like long polling?

**Review by radeknovis — APPROVED** [review]

File: Example/IntegrationTests/XPlatform/Web3Wallet/XPlatformW3WTests.swift (hunk 1 above)
Comment (llbartekll) [reply Example/IntegrationTests/XPlatform/Web3Wallet/XPlatformW3WTests.swift:104]: 👍 good idea, 100%, will speak with js team

**Review by joe10832 — COMMENTED** [review]
@radeknovis

**Review by joe10832 — COMMENTED** [review]
@joe10832

**Pull request merged**